bin/lib/create.js

```diff
@@ -225,11 +225,6 @@ function validateProjectName (project_name) {
         return Q.reject(new CordovaError(msg + 'Project name cannot be CordovaActivity'));
```

Comment:
If this name is not used as a class name, then this check would probably be superfluous too, right? And that would mean this whole function basically degenerates to a check if `project_name` is truthy. Which means it could be removed altogether.

_Note_: I'm just extrapolating based on the information you gave in the PR description.

Reply:
You're probably right there. I'll do some testing with that removed.

Reply:
Great! :+1:

Reply:
Removed that condition, kept the empty name check as that is still a perfectly valid check. Still able to build and run the sample cordova app. :+1: